README.md

```diff
@@ -79,7 +79,7 @@ package` and push it with `mvn deploy`.  Of course you can also say
 A corresponding `Dockerfile` could look like:
 
 ```
-FROM dockerfile/java:oracle-java8
+FROM java:8
```

Comment:
This image/repo seems to be deprecated and no longer updated: https://hub.docker.com/_/java/

Does `dockerfile/java` not work for some reason?

Reply:
Hi Matt, thank your for your response!

I was not able to pull dockerfile/java, nor have I found a hint in the repo on how to build it. Might be an internal resource of spotify?

```
docker pull dockerfile/java
Using default tag: latest
Error response from daemon: pull access denied for dockerfile/java, repository does not exist or may require 'docker login'
```

But you are right, the java:8 repo is deprecated, sorry for that. Would you agree if I changed it to *openjdk:8-jre* ?

Reply:
ah, maybe that is deprecated too. The repo is at https://github.com/dockerfile/java but the Docker Hub page is a 404 (https://hub.docker.com/r/dockerfile/java/)

Since it is just an example changing to openjdk-8-jre makes sense to me 👍

Reply:
Ok, I have replaced the FROM directive in the dockerfile. I used it to pack a Spring Boot application, and it runs perfectly!